kythe/extractors/gcp/examples/gradle.yaml

```diff
@@ -24,21 +39,29 @@ steps:
     - '-b'
     - '/workspace/code/build.gradle'
   env:
-    - 'KYTHE_CORPUS=gradlepoctest'
+    - 'KYTHE_CORPUS=${_CORPUS}'
     - 'KYTHE_OUTPUT_DIRECTORY=/workspace/out'
-    - 'KYTHE_OUTPUT_FILE=/workspace/out/test.kzip'
     - 'KYTHE_ROOT_DIRECTORY=/workspace/code'
     - 'JAVAC_EXTRACTOR_JAR=/opt/kythe/extractors/javac_extractor.jar'
     - 'REAL_JAVAC=/usr/bin/javac'
     - 'TMPDIR=/workspace/out'
     - 'KYTHE_JAVA_RUNTIME_OPTIONS=-Xbootclasspath/p:/opt/kythe/extractors/javac9_tools.jar'
   volumes:
     - name: 'kythe_extractors'
-      path: '/opt/kythe/extractors/'
+      path: '/opt/kythe/extractors'
+  id: 'EXTRACT0'
```

Comment:
Curious why the steps are named "XYZ0" and not "XYZ"?

Reply:
I don't have special casing logic differentiating single-extract vs multiple-extract for naming steps.  So this is required for extracting e.g. multiple different poms that don't link in a single top-level action.  I could have logic that omits the 0 for single case, but not sure if it's worth doing.

Reply:
If we feel strongly about having different logic there, we can do so.  It wouldn't be too messy I think.  And the majority of cases will probably have just one target to extract (I'd guess, could be wrong), so it would be cleaner output.

Reply:
ok see #3526 which special cases the default single-target to not have an index here.